lib/esConnector.js

```diff
@@ -728,46 +732,23 @@ ESConnector.prototype.all = function all(model, filter, done) {
     var idName = self.idName(model);
     log('ESConnector.prototype.all', 'idName', idName);
 
-    if(filter && filter.suggests) { // TODO: remove HACK!!!
-        self.db.suggest(
-            self.buildFilter(model, idName, filter, self.defaultSize)
-        ).then(
-            function (body) {
-                var result = [];
-                if (body.hits) {
-                    body.hits.hits.forEach(function (item) {
-                        result.push(self.dataSourceToModel(model, item, idName));
-                    });
-                }
-                log('ESConnector.prototype.all', 'model', model, 'result', JSON.stringify(result,null,2));
-                done(null, result);
-            }, function (err) {
-                console.trace(err.message);
-                if (err) {
-                    return done(err, null);
-                }
-            }
-        );
-    }
-    else {
-        self.db.search(
-            self.buildFilter(model, idName, filter, self.defaultSize)
-        ).then(
-            function (body) {
-                var result = [];
-                body.hits.hits.forEach(function (item) {
-                    result.push(self.dataSourceToModel(model, item, idName));
-                });
-                log('ESConnector.prototype.all', 'model', model, 'result', JSON.stringify(result,null,2));
-                done(null, result);
-            }, function (err) {
-                console.trace(err.message);
-                if (err) {
-                    return done(err, null);
-                }
+    self.db.search(
```

Comment:
@aquid 
It is deprecated since 5.0
https://www.elastic.co/guide/en/elasticsearch/reference/5.2/search-suggesters.html

And anyways, suggests work through search if you take a look at the docs:
https://www.elastic.co/guide/en/elasticsearch/reference/2.3/search-suggesters.html
https://www.elastic.co/guide/en/elasticsearch/reference/1.7/search-suggesters.html
They only mention that this endpoint exists ...